server/src/main/java/io/crate/metadata/doc/DocTableInfoFactory.java

```diff
@@ -259,7 +259,6 @@ public DocTableInfo create(RelationName relation, Metadata metadata) {
             clusteredBy,
             concreteIndices,
             concreteOpenIndices,
-            numberOfShards,
```

Comment:
Could you please check if we can remove `numberOfShards` completely from `create()`, I think it's only used to pass it to the ctor.